### PR TITLE
bump inkwell to 0.1.0

### PR DESCRIPTION
**Pull request by hamidrezakp**
Bumped `inkwell` dependency to `0.1.0`

Also i would like to know if you have any plans to use `llvm-sys` with latest version or not, i can try to refactor the code.

### CODE REVIEW

File: lib/compiler-llvm/src/translator/intrinsics.rs
Comment (syrusakbary) [inline lib/compiler-llvm/src/translator/intrinsics.rs:1978]: We can just delete this

**Review by syrusakbary — APPROVED** [review]
Looks good to me, great work!

File: lib/compiler-llvm/src/translator/intrinsics.rs
Comment (hamidrezakp) [reply lib/compiler-llvm/src/translator/intrinsics.rs]: ops, fixed!

**Comment by hamidrezakp** [comment]
> Looks good to me, great work!

Thank you!

**Comment by hamidrezakp** [comment]
Is there anything needed from my side to make this PR merged?

**Comment by ptitSeb** [comment]
> Is there anything needed from my side to make this PR merged?

Nothing on your side. We have some issues with github actions not triggering here.

**Comment by syrusakbary** [comment]
After some days we have no idea on why tests are not being triggered.
To keep things going, I'll merge and see if tests pass in master. If they don't, we can revert the merge afterwards

**Comment by syrusakbary** [comment]
@hamidrezakp we had to revert this PR in #3563 because a bug in inkwell. Once that's resolved we should be good to merge it again

**Pull request merged**
